app/messages/index.js

```diff
@@ -0,0 +1,27 @@
+import util from 'util'
+import { messageConfig } from '../config/index.js'
+import { MessageReceiver } from 'ffc-messaging'
+import { farmerApply } from './farmer-apply.js'
+
+const handleMessage = async (message, receiver) => {
+  try {
+    console.log('Received message: ', message.body)
+    await farmerApply(message)
+    await receiver.completeMessage(message)
+  } catch (error) {
+    throw new Error('Message error', util.inspect(error.message, false, null, true))
+  }
+}
+
+const startMessaging = async () => {
```

Comment:
I'd swap the order of these functions so it reads more naturally.

ie this is hte main function of this module, the other is referenced by it.

Think like a newspaper, more important to least important.

Reply:
Done.